experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -1231,12 +1232,41 @@ describe("FormAudience", () => {
   });
 });
 
-it("disables fields when locked", async () => {
+it("fields should be enabled for draft experiments", async () => {
   render(
     <Subject
       experiment={{
         ...MOCK_EXPERIMENT,
         application: NimbusExperimentApplicationEnum.FENIX,
+        isRollout: false,
+        status: NimbusExperimentStatusEnum.DRAFT,
+        targetingConfig: [
+          {
+            label: "No Targeting",
+            value: "",
+            applicationValues: [
+              NimbusExperimentApplicationEnum.DESKTOP,
+              "TOASTER",
+            ],
+            description: "No targeting configuration",
+            stickyRequired: false,
+            isFirstRunRequired: false,
+          },
+        ],
+      }}
+    />,
+  );
+  expect(screen.getByTestId("isSticky")).toBeEnabled();
+  expect(screen.getByTestId("firefoxMinVersion")).toBeEnabled();
+  expect(screen.getByTestId("channel")).toBeEnabled();
+});
+
+it("enables fields and buttons for live rollouts", async () => {
```

Comment:
you can add one more test for field disable for live experiment,

Reply:
we already have test covered, my bad

Reply:
We are only showing the Audience button [for rollouts](https://github.com/mozilla/experimenter/blob/30acc0af7ebf8864b0258c6e2c01657a60f5b709/experimenter/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.tsx#L195), so there is no way that a live experiment can get this far. That functionality is already tested in https://github.com/mozilla/experimenter/pull/8136 👍